src/apps/reservation-list/modal/reservation-details/helper.ts

```diff
@@ -0,0 +1,34 @@
+import { ComplexSearchWithPaginationWorkAccessQuery } from "../../../../core/dbc-gateway/generated/graphql";
+
+type AccessManifestations =
+  ComplexSearchWithPaginationWorkAccessQuery["complexSearch"]["works"][0]["manifestations"]["all"];
+
+export const findAccessManifestationByIdentifier = (
+  manifestations: AccessManifestations,
+  identifier: string
+) => {
+  return manifestations.filter((manifestation) =>
+    manifestation.identifiers.find(
+      (id) => id.value.replace(/\D/g, "") === identifier.replace(/\D/g, "")
```

Comment:
You are stripping the faust ids away and comparing the work-of:* strings? 
I think it would be nice if you either:
* Write a comment about what you are comparing
* Write the regex more explicitly